script.js

```diff
@@ -1,4 +1,51 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var computerNumber = randomNumber();
+  var computerChoice = "";
+  console.log("compter number is supposed to be " + computerNumber);
+  var lowercase = input.toLowerCase();
+  console.log("Uppercase and Lowercase does not matter");
+
+  if (computerNumber == 1) {
+    computerChoice = "scissors";
+  }
+
+  if (computerNumber == 2) {
+    computerChoice = "paper";
+  }
+
+  if (computerNumber == 3) {
+    computerChoice = "stone";
+  }
+
+  var myOutputValue =
+    "The computer chose " +
+    computerChoice +
+    ". <br> You chose " +
+    lowercase +
+    "<br><br> You lose! Bummer." +
+    "<br><br> Hey, dont stop here keep going! Try again.";
+  console.log("game results ");
+
+  // Add input validation to check if what the user types is a valid choice, and if not, kindly let the user know that there are only 3 input options, and ask them to try again.
+  if (input != "scissors" && input != "paper" && input != "stone") {
+    myOutputValue =
+      "Invalid Input!" +
+      "<br><br> There are only 3 options Scissors, Paper, or Stone." +
+      "<br><br> Please try again";
+    console.log("If the player does not enter valid input");
+  }
```

Comment:
Great job on figuring out the input validation logic! Plenty of students can mess up here, where they use `||` OR instead of `&&` AND, and it doesn't work.

A little _advanced_ cheatsheet for you (for subsequent projects, or programming classes) if you ever get tripped by when to use `&&` or `||`:

For `if (subcondition1 || subcondition2 || subcondition3)`, the entirety of `(subcondition1 || subcondition2 || subcondition3)` will be `true` **as long as just 1 (or more)** of `subcondition1 `or `subcondition2` or `subcondition3` is true.

For `if (subcondition1 && subcondition2 && subcondition3)`, the entirety of `(subcondition1 && subcondition2 && subcondition3)` will be `true` **only if all** of `subcondition1` and `subcondition2` and `subcondition3` are `true`.